lib/public/views/Logs/Logs.js

```diff
@@ -50,14 +55,89 @@ export default class Overview extends Observable {
         this.notify();
     }
 
+    /**
+     * Create the log with the variables set in the model, handle errors appriopately
+     * @returns {undefined}
+     */
+    async createLog() {
+        const response = await fetchClient('api/logs', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                title: this.getTitle(),
+                text: this.getText(),
+            }),
+        });
+
+        const result = await response.json();
+
+        if (result.data) {
+            const logId = await result.data.id;
```

Comment:
Are property getters async?

Reply:
This was residue code from debugging the automatic redirect

Reply:
Fixed